### ISSUE TITLE
[med] JAWS+Chrome: Many Pages: Multiple buttons announced without labels

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5519]
## Action Performed:
Prerequisite: the user is logged in.
1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Press Tab key to focus the 'My settings' button and press Space key.
3. Press Tab key to focus the button of profile icon and listen to the announcement.

## Expected Result:
The button should be announced with a meaningful label, e.g.: 'profile'.

## Actual Result:
The button has no label announced, it announces the label and role of the previously focused element: 'Close button'.

HTML:
`<div tabindex="0" class="css-1dbjc4n r-1loqt21 r-1otgn73" style="margin-bottom: 12px;"><div class="css-1dbjc4n" style="height: 80px; width: 80px;"><div class="css-1dbjc4n r-633pao"><div class="css-1dbjc4n r-1mlwlqe r-1udh08x r-417010" style="border-radius: 40px; height: 80px; width: 80px;"><div class="css-1dbjc4n r-1niwhzg r-vvn4in r-u6sd8q r-4gszlv r-1p0dtai r-1pi2tsx r-1d2f490 r-u8s1d r-zchlnj r-ipm5af r-13qz1uu r-1wyyakw" style="background-image: url(&quot;https://d2k5nsl2zxldvw.cloudfront.net/images/avatars/avatar_5.png&quot;
);"></div><img alt="" draggable="false" src="https://d2k5nsl2zxldvw.cloudfront.net/images/avatars/avatar_5.png"
class="css-9pa8cd"></div></div><div class="css-1dbjc4n r-1awozwy r-1777fci r-u8s1d" style="background-color: rgb(3, 212, 124); border-color: rgb(255, 255, 255); border-radius: 8px; border-width: 2px; bottom: 4px; height: 16px; right: 4px; transform: rotate(0deg) scale(1); width: 16px; z-index: 10;"></div></div></div>`

Other occurrences:
1. On Preferences overlay the button 'Back' announces as 'chat left parent loading action right parent button'
2. On Chat page the buttons 'Pin' and 'Call' are announced with no label.

## Workaround:
No workaround

## Area issue was found in:
Settings overlay, Preferences overlay, Chat page

## Failed WCAG checkpoints
4.1.2

## User impact:
The screen reader users are not aware of the purpose of these elements.

## Suggested resolution:
Once the element is implemented as a button (see issue 5179534), use the aria-label attribute to announce the button with a meaningful label (e.g.: 'profile'). See: https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA14.html

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/134737400-04c0dc9c-88e2-4665-be77-0acc166f22d6.mp4

<img width="1323" alt="Bug5182757_WCAC_failure_4 1 2 _Profile_button_was_announced_as_close_button" src="https://user-images.githubusercontent.com/88733897/134737422-30a61778-9344-4603-95a0-4db1e176944c.PNG">
![Bug5182757_Chat-Pin and Call buttons announced without labels](https://user-images.githubusercontent.com/88733897/134737431-69283332-3506-4052-929d-e2da7d7f97f3.png)
![Preferences overlay - Back button announcement](https://user-images.githubusercontent.com/88733897/134737606-37c128c0-3026-4c57-9be5-f6524f126d7d.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
